From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tri Vo <trong@google.com>
Date: Tue, 14 May 2019 17:32:43 -0700
Subject: ANDROID: gki_defconfig: disable CONFIG_LCD_CLASS_DEVICE module

CONFIG_LCD_CLASS_DEVICE is set to "m" by default when
CONFIG_BACKLIGHT_LCD_SUPPORT is set. We don't need lcd.ko though.
---
 arch/arm64/configs/gki_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index cc9ecd83a69c..4b17f0f9d0ae 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -284,6 +284,7 @@ CONFIG_DRM=y
 # CONFIG_DRM_FBDEV_EMULATION is not set
 CONFIG_DRM_VIRTIO_GPU=y
 CONFIG_BACKLIGHT_LCD_SUPPORT=y
+# CONFIG_LCD_CLASS_DEVICE is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_SOUND=y
 CONFIG_SND=y
